src/App.js

```diff
@@ -3,11 +3,25 @@ import 'bootstrap/dist/css/bootstrap.min.css';
 import './App.css';
 import {Nav, Navbar, Button} from 'react-bootstrap';
 import logo from './logo.png';
-
+import Header from './Header/Header';
+import Emulator from './Emulator/Emulator';
 
 // rename App.js and App.css to navbar component
 // Move navbar.js, navbar.css, and logo(?) once file structure is determined
 
+const data = {
+  "id": 1,
+  "module_number": 1,
+  "module_name": "Local Area Networks",
```

Comment:
submodule? (I think it's possible that at this depth we don't care which module it belongs to)

Reply:
I can definitely change the names of the hardcoded JSON response. Should `module_number` and `module_name` both be changed to `submodule_*`?

Reply:
I say yes 👍  I think it's safer for us to error on the side of specific over general in so future debugging is more intuitive.